service/src/main/resources/application.yml

```diff
@@ -32,7 +32,7 @@ drshub:
       accessMethodConfigs:
         - type: gs
           auth: fence_token
-          fetchAccessUrl: false
+          fetchAccessUrl: true
     crdc:
```

Comment:
Wasn't this ticket to do CRDC too?

Reply:
Yeah sorry I was looking into this at the very end of the day yesterday and got into the weeds with trying to make sure these configs match and then forgot to actually say what I was doing. But long story short, I was right the first time and yes this should be updated.